ivy/functional/frontends/torch/non_linear_activation_functions.py

```diff
@@ -19,6 +19,13 @@ def softmax(input, dim=None, dtype=None):
 
 
 def gelu(input, approximate='none'):
+    assert approximate in ['none','tanh']
+    
+
+    if approximate == 'none:
+        approximate = False
+    elif approximate == 'tanh':
```

Comment:
I suppose you can just have an else statement here to make things easier.

Reply:
this afternoon I'm writing on my phone, because I'm on outside so I code using github IDE I'm sorry

Reply:
That is why there are many strange mistakes. Understandable.